content/javascript/concepts/dates/terms/getUTCFullYear/getUTCFullYear.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: '.getUTCFullYear()'
+Description: 'Returns the year of the specified date according to universal time.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.getUTCFullYear()`** method is a built-in function in JavaScript that returns the year of the specified date according to universal time. It extracts the four-digit year (e.g. 2023) from a given `Date` object, based on the Coordinated Universal Time (UTC) time zone.
+
+This method is particularly useful when you need to retrieve the year component of a date regardless of the time zone or local system settings. It ensures consistent results regardless of the user's geographical location.
+
+## Syntax
+
+```pseudo
+const date = new Date();
+date.getUTCFullYear();
+```
+
+A `Date` object is used for working with dates and times. The object provides a way to create, manipulate, and format dates in various formats.
+
+A `Date` object can be created by declaring a new variable with the `new Date()` constructor.
+
+## Example
+
+Consider the following example that demonstrates the usage of the `.getUTCFullYear()` method:
+
+```js
+const date = new Date('2023-06-06T10:30:00Z');
+const year = date.getUTCFullYear();
+
+console.log(year); // Output: 2023
```

Comment:
Rather than a comment in the code, can we have a ` ```shell ` block illustrating the output? Something like the following:

````
This example results in the following output:

```shell
2023
```
````